lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -217,13 +217,13 @@ sub schedule {
 
     # Exit only when database state is consistent.
     if ($quit) {
-        log_debug("Exiting");
+        log_error("Exiting");
         exit(0);
     }
 
     my @allocated_jobs;
 
-    log_debug("-> Scheduling new jobs.");
+    log_info("-> Scheduling new jobs.");
```

Comment:
Isn't that one of the lines which are printed (annoyingly) often? If so, I wouldn't make it `log_info`.

Reply:
no, this is one of the lines that are printed once per `schedule()` call

Reply:
... which happens annoyingly often in fullstack test

Reply:
@coolo yeah, but it's good enough, the ammount of occurences is not as often as other messages. As a last resort we can log the full stack test's scheduler to a separate file, and cat it along the rest when there's a failure